backend/src/main/java/com/map/repository/UserRepository.java

```diff
@@ -1,40 +1,45 @@
 package com.map.repository;
 
-import com.map.configuration.DatabaseConnection;
-import com.map.dto.UserDto;
 import com.map.model.User;
-import lombok.AllArgsConstructor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Repository;
 
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
 @Repository
-@AllArgsConstructor
 public class UserRepository {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(UserRepository.class);
-    private final DatabaseConnection databaseConnection;
+    private static final Logger LOG = LoggerFactory.getLogger(UserRepository.class);
+    private Connection connection;
+
+    public UserRepository(@Value("${spring.datasource.url}") String url,
```

Comment:
It doesn't look pretty :) 

`@Value("${spring,datasource.url}")
private String url;`
etc. could be more readable ?